src/kagglehub/logger.py

```diff
@@ -39,8 +39,8 @@ def _configure_logger(log_dir: Optional[Path] = None) -> None:
         handler = library_logger.handlers.pop()
         while handler.filters:
             handler.filters.pop()
-    logging_enabled = os.environ.get(KAGGLE_LOGGING_ENABLED_ENV_VAR_NAME, False)
-    if str(logging_enabled).lower() in ("true", 1):
+    logging_enabled = os.environ.get(KAGGLE_LOGGING_ENABLED_ENV_VAR_NAME, "false")
+    if logging_enabled.lower() in ("true", 1):
```

Comment:
should it be: `("true", "1")`? Looks like we had a bug and it wouldn't work if people set the env var to "1": https://screenshot.googleplex.com/57yk5xHdVnxe7pT